extensions/amp-social-share/1.0/social-share.js

```diff
@@ -52,23 +52,31 @@ export function SocialShare(props) {
 
   const type = props['type'].toUpperCase();
   const baseStyle = CSS.BASE_STYLE;
-  const backgroundStyle = CSS[type] || CSS.DEFAULT;
+  const backgroundStyle = CSS[type];
   const size = {
     width: checkedWidth,
     height: checkedHeight,
   };
+
+  // Only show children if not using a pre-configured type
+  const children = getSocialConfig(props['type']) ? (
+    <SocialShareIcon
+      style={{...backgroundStyle, ...baseStyle, ...size}}
```

Comment:
TIL `{...undefined}` resolves to `{}`. Neat!

Reply:
woohoo!